04-kogito-visas/README.md

```diff
@@ -113,6 +111,34 @@ To run the generated native executable, generated in `target/`, execute
 ./target/kogito-visas-{version}-runner
 ```
 
+### Start Kogito Data Index Service
+
+We have to start the Data Index Service, we can find this service description more detailed at [Kogito Data Index Service](https://github.com/kiegroup/kogito-runtimes/wiki/Data-Index-Service).
```

Comment:
Same here point people to maven repo and run the data index ‘java -jar runner’ with the proto folder property